Meme-ify-Me/app/src/main/res/layout/activity_vanilla_meme_edit.xml

```diff
@@ -0,0 +1,161 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    tools:context="nyc.c4q.rosmaryfc.meme_ify_me.VanillaMemeEdit">
+
+    <!-- meme preview -->
+    <LinearLayout
+        android:id="@+id/meme_preview"
+        android:layout_weight="2"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/successkid">
```

Comment:
Sorry it is too late but just as memo, the line 19 needs to be changed because it is only taking the demo (default preview) picture right now.  The chosen image (camera or from gallery) should be placed as a image here.